### ISSUE TITLE
A question about..

### ISSUE THREAD

**Issue by foo123** [RedactedProfile/GraphQLMovieDB#1]
[PasswordlessBundle](https://github.com/RedactedProfile/PasswordlessBundle), since I cannot post a question there as it is archived. Are there any plans this can be used with latest symfony (4)? I see the code is in a state where various things are set as TODO (eg configuration) so I see it is not fuinished and it is now archoved. 

Was it working good with Symfony2? Do you think it worths the effort (I may undertake it myself) to try to update the bundle for Symfony 4 as I need it for a project that uses magic link authentication?

Thanks,
Nikos

**Comment by RedactedProfile** [comment]
Hi @foo123 thanks for reaching out

The project this was initially developed for ended up deciding against any form of user system, so it was never brought to completion. 

If I remember the project correctly, all the pieces were mostly in place, but I got hung up on Symfony 2's token system, as at this time and stage it was quite difficult to get around and without a password present (even when supplying my own authenticator) the symfony authentication token simply would not set. 

Since then, Symfony 3 had been released, and then the Guard project implemented, and now Symfony 4. The work of dealing with Symfony's user and firewall system is greatly reduced and I imagine it wouldn't be too difficult anymore to implement a passwordless system. 

As much as I would love for this bundle to be the initial root of an updated passwordless bundle for symfony 4, the truth is a tremendous deal of Symfony has changed since this bundle was created and since archived.  I would start a new bundle targeting Symfony 4 from scratch. 


Cheers and good luck @foo123 :)
